### DISCUSSION TITLE
Upgrade from 5.12.1 to 6.0.2

### DISCUSSION THREAD

**Discussion by jbrabant** [ServiceStack/Discuss#8] [General]
I was wondering if i'll get into trouble if I upgrade from 5.12.1 (which I had got from MyGet a while ago) to 6.0.2
Can I stay on 4.7.2 without having to install any .NET6 (core) ?

**Comment by mythz** [comment]
Yep .NET Framework v4.7.2 continues to have first class support in v6+.

Only .NET Framework v4.5 support was dropped, by upgrading and replacing all net45 builds with net472 builds.

**Comment by jbrabant** [comment]
Got some issues and I do not have time right now to go through them.  We are at the end of the project.
I am not sure why, but I think I got 5.12.1 from your MyGet repo and now I can't find it anywhere.
My NuGet restore fails not finding it...Is it possible to make it available?
If not, is it pretty safe to update to 5.14?

Thanks for your help.

**Comment by mythz** [comment]
You can find the versioning scheme for NuGet and MyGet pre-release packages at: https://docs.servicestack.net/myget#versioning-scheme

MyGet only contains the latest version which ends in an **odd** patch number, e.g. **v5.12.1** and only lasts until the next release on NuGet so they should only be referenced by wildcard, e.g. `v6.*`.

Each version ending in an **even** patch number like **v5.13.2** or **v5.14.0** are immutable packages published to NuGet, they should be used instead if you want officially released immutable packages.
